From f594e8d33e6d74ea8b8a1c8f354cab1cc8ecae4d Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 30 Jan 2025 09:53:03 +0100
Subject: [PATCH] arm64: dts: st: remove power domains from sai parent nodes on
 stm32mp211

The pm runtime is managed at child nodes level only.
So remove the power-domains properties from SAI parent nodes
in stm32mp211.dtsi.

Change-Id: I8cc6dd11f5301efeb8dd34d90f27a2862ac091a3
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/432940
Tested-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Eric FOURMONT <eric.fourmont-ext@st.com>
---
 arch/arm64/boot/dts/st/stm32mp211.dtsi | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/arch/arm64/boot/dts/st/stm32mp211.dtsi b/arch/arm64/boot/dts/st/stm32mp211.dtsi
index 8d1db320e585..32e1e9dd26a7 100644
--- a/arch/arm64/boot/dts/st/stm32mp211.dtsi
+++ b/arch/arm64/boot/dts/st/stm32mp211.dtsi
@@ -1236,7 +1236,6 @@ sai1: sai@40290000 {
 				interrupts = <GIC_SPI 140 IRQ_TYPE_LEVEL_HIGH>;
 				resets = <&rcc SAI1_R>;
 				access-controllers = <&rifsc 49>;
-				power-domains = <&d1_pd>;
 				status = "disabled";
 
 				sai1a: audio-controller@40290004 {
@@ -1273,7 +1272,6 @@ sai2: sai@402a0000 {
 				interrupts = <GIC_SPI 145 IRQ_TYPE_LEVEL_HIGH>;
 				resets = <&rcc SAI2_R>;
 				access-controllers = <&rifsc 50>;
-				power-domains = <&d1_pd>;
 				status = "disabled";
 
 				sai2a: audio-controller@402a0004 {
@@ -1310,7 +1308,6 @@ sai3: sai@402b0000 {
 				interrupts = <GIC_SPI 164 IRQ_TYPE_LEVEL_HIGH>;
 				resets = <&rcc SAI3_R>;
 				access-controllers = <&rifsc 51>;
-				power-domains = <&d1_pd>;
 				status = "disabled";
 
 				sai3a: audio-controller@402b0004 {
@@ -1361,7 +1358,6 @@ sai4: sai@40340000 {
 				interrupts = <GIC_SPI 187 IRQ_TYPE_LEVEL_HIGH>;
 				resets = <&rcc SAI4_R>;
 				access-controllers = <&rifsc 52>;
-				power-domains = <&d1_pd>;
 				status = "disabled";
 
 				sai4a: audio-controller@40340004 {
-- 
2.50.1

